ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/deneb/block/BlockProcessorDeneb.java

```diff
@@ -98,6 +101,15 @@ public NewPayloadRequest computeNewPayloadRequest(final BeaconBlockBody beaconBl
     return new NewPayloadRequest(executionPayload, versionedHashes);
   }
 
+  @Override
+  public Bytes32 computeVoluntaryExitDomain(
+      final SignedVoluntaryExit signedVoluntaryExit, final BeaconState state) {
+    return miscHelpers.computeDomain(
+        Domain.VOLUNTARY_EXIT,
+        specConfig.toVersionCapella().orElseThrow().getCapellaForkVersion(),
```

Comment:
Just a suggestion in meantime to change this to `SpecConfigDeneb.required(specConfig).getCapellaForkVersion()` (clearer error message)